src/app/components/table/table.tsx

```diff
@@ -97,7 +93,7 @@ const TableComponent = <Data extends object>(props: ITableProps<Data>) => {
                     <th
                       key={keyTh}
                       {...restColumn}
-                      style={{...style, cursor: (!column.disableSortBy && !disabled) ? "default" : "pointer", width: column.width, textAlign: (centerHeader) ? "center": "left"}}
+                      style={{...style, cursor: (column.disableSortBy || disabled) ? "default" : "pointer", width: column.width, textAlign: (centerHeader) ? "center": "left"}}
```

Comment:
Nice use of De Morgan's Laws :)

https://en.wikipedia.org/wiki/De_Morgan%27s_laws